### PR TITLE
Kd okta 349558 investigate failing tests

### PR DESCRIPTION
**Pull request by KostiantynDrozd-okta**
Uncommented tests and fixed failed tests.

### CODE REVIEW

File: scripts/e2e.sh [hunk 1]
```diff
@@ -12,21 +12,23 @@ export TRAVIS=true
...
-    exit 0;
-    # npm run pretest
-    # npm run test:okta-hosted-login
-    # kill -s TERM $(lsof -t -i:8080 -sTCP:LISTEN)
-    # npm run test:custom-login
+    # exit 0;
+    npm run pretest
+    npm run test:okta-hosted-login
+    kill -s TERM $(lsof -t -i:8080 -sTCP:LISTEN)
+    kill -s TERM $(lsof -t -i:8000 -sTCP:LISTEN)
```
Comment (swiftone) [inline scripts/e2e.sh:30]: Why is this here twice?

File: scripts/e2e.sh [hunk 2]
```diff
@@ -12,21 +12,23 @@ export TRAVIS=true
...
+    npm run test:custom-login
```
Comment (swiftone) [inline scripts/e2e.sh:31]: Why do we have to kill after the okta-hosted test, but not here?

**Review by swiftone — APPROVED** [review]
See questions

File: scripts/e2e.sh (hunk 1 above)
Comment (KostiantynDrozd-okta) [reply scripts/e2e.sh:30]: @swiftone 
There are different ports, 8080 and 8000 :)

File: scripts/e2e.sh (hunk 2 above)
Comment (KostiantynDrozd-okta) [reply scripts/e2e.sh:31]: @swiftone 
'But not here' - you mean after custom login?

Comment (KostiantynDrozd-okta) [reply scripts/e2e.sh:31]: Because ports are still busy after okta-hosted-login tests, and custom-login tests are failing.

File: scripts/e2e.sh (hunk 1 above)
Comment (swiftone) [reply scripts/e2e.sh:30]: I am blind

File: scripts/e2e.sh (hunk 2 above)
Comment (swiftone) [reply scripts/e2e.sh:31]: But will the server clean itself up after the custom-login tests are complete?

Comment (KostiantynDrozd-okta) [reply scripts/e2e.sh:31]: @swiftone 
I checked this locally, and it looks, that all ports are free after custom-login tests are complete.

Comment (swiftone) [reply scripts/e2e.sh:31]: 👍
